support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx

```diff
@@ -61,25 +70,24 @@ import {
 	updateLastName,
 	updatePaymentMethod,
 } from 'pages/contributions-landing/contributionsLandingActions';
+import type { Action } from 'pages/contributions-landing/contributionsLandingActions';
 import type {
 	State,
-	Stripe3DSResult,
 	StripePaymentRequestButtonData,
 } from 'pages/contributions-landing/contributionsLandingReducer';
-import type { PaymentMethod } from 'helpers/forms/paymentMethods';
-import { Stripe } from 'helpers/forms/paymentMethods';
-import type { StripeAccount } from 'helpers/forms/stripe';
 import type { Csrf as CsrfState } from '../../../../helpers/csrf/csrfReducer';
 import type { LocalCurrencyCountry } from '../../../../helpers/internationalisation/localCurrencyCountry';
 import { trackComponentEvents } from '../../../../helpers/tracking/ophan';
-import { Button } from '@guardian/src-button';
 // ----- Types -----//
-type PaymentRequestObject = Record<string, any>; // Just to make it clearer when we're passing this object around
 
-/* eslint-disable react/no-unused-prop-types */
+type PaymentRequestData = Omit<
+	PaymentRequestPaymentMethodEvent,
+	'complete' | 'paymentMethod'
+>;
```

Comment:
This is a bit of a strange type, but we destructure a `PaymentRequestPaymentMethodEvent` like

```ts
{ complete, paymentMethod, ...data } 
```

and then pass it around quite a lot. This felt like the most convenient way to type it.

Reply:
I think I see what's happening here, would the alternative be to just pass around the original stripe type?
```
(paymentMethodEvent: PaymentRequestPaymentMethodEvent) => {
  const { complete, paymentMethod } = paymentMethodEvent;
  ...
}
```

Reply:
Yeah that's a good idea!